packages/tc-ui/src/primitives/temporal/server.jsx

```diff
@@ -25,19 +17,22 @@ const formatDateTime = (value, type) => {
 	}
 
 	if (type === 'Time') {
-		const timeInput = value;
-		return formatTime(timeInput);
+		return formatValue(`2020-01-15T${value}`, 'h:mm:ss a');
```

Comment:
This is a bit confusing - if you used moment you can just do 
`var time = moment('12:15:33', 'HH:mm:ss a');` without having to add a date and convert into ISOstring but not sure its worth the change

Reply:
I agree moment is a bit simpler, but it is notorious for being really big, which is why I avoid it.

Although I just checked out their website and it looks like the latest version is much smaller.

Reply:
Yea I saw that when I compared the two which was surprising. Although I guess this works for now so..?

Reply:
I had a go at using moment. For now I'm gonna pass as I couldn't get it to work and don't have the time to explore more. I'll leave a comment in the code to explain why we add eth date though as could benefit from an explanation